src/stream.rs

```diff
@@ -55,14 +55,18 @@ pub struct OutputStreamBuilder {
     config: OutputStreamConfig,
 }
 
-impl Default for OutputStreamConfig {
-    fn default() -> Self {
-        Self {
-            channel_count: 2,
-            sample_rate: HZ_44100,
-            buffer_size: BufferSize::Default,
-            sample_format: SampleFormat::I16,
-        }
+impl core::fmt::Debug for OutputStreamBuilder {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
I do not know why these changes are new here? they come from main. Its not a problem though I do not mind.